app/main/routes_zseries_api.py

```diff
@@ -0,0 +1,591 @@
+import json, uuid, os
+from datetime import datetime
+from pathlib import Path
+from time import mktime
+from flask import *
+from flask_socketio import emit
+from webargs import fields
+from webargs.flaskparser import use_args, FlaskParser
+from . import main
+from .routes_frontend import get_zseries_recipes, load_brew_sessions_by_machine
+from .. import *
+from enum import Enum
+from random import seed
+from random import randint
+
+
+arg_parser = FlaskParser()
+seed(1)
+
+events = {}
+
+latest_firmware = {
+    "version": "0.0.119",
+    "source": "https://picobrewcontent.blob.core.windows.net/firmware/zseries/zseries_0_0_119.bin"
+}
```

Comment:
@chiefwigms what do you think about pulling each and every firmware down from the Azure backup we have and supporting in the config.yml a way to specify the "required firmware version" so we can lock devices to specific known copies of the firmware?